pkg/node/types/node.go

```diff
@@ -171,6 +173,9 @@ type Node struct {
 
 	// Node labels
 	Labels map[string]string
+
+	// NodeIdentity is the numeric identity allocated for the node
+	NodeIdentity uint32
```

Comment:
Why have differing int sizes? Can we just have both be `uint32` or `uint64`? That'll make it easier to reason about because it's confusing to someone at first glace why they're different. Also avoids useless casts between types.

Reply:
Historial reasons, I presume. Size changes would be API breaking (on existing APIs). Here I'm just going with the conventions we already have.